### ISSUE TITLE
Can't install it in Ubuntu 20.04

### ISSUE THREAD

**Issue by melodie11** [thenaterhood/gscreenshot#101]
Hello,
Following the README, I don't know how to install it with pip. This is what happens:

```
sudo pip install -e
sudo: pip : command not found

sudo pip3 install -e
Usage:   
  pip3 install [options] <requirement specifier> [package-index-options] ...
  pip3 install [options] -r <requirements file> [package-index-options] ...
  pip3 install [options] [-e] <vcs project url> ...
  pip3 install [options] [-e] <local project path> ...
  pip3 install [options] <archive url/path> ...

-e option requires 1 argument
```
So which file from the sources directory should I give it as argument? 

PS: trying the other command line, and no luck either:
```
sudo python setup.py install
Traceback (most recent call last):
  File "setup.py", line 3, in <module>
    from setuptools import setup
ImportError: No module named setuptools
```
(I do have python3-setuptools installed in my system, in case you'd wonder… )

**Comment by thenaterhood** [comment]
The current directory, as in "."

"sudo pip install -e ."

--

**Comment by melodie11** [comment]
Thank you very much! I tried right away. However, no more luck:

`$ sudo pip3 install -e .
ERROR: Exception:
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/pip/_internal/cli/base_command.py", line 186, in _main
    status = self.run(options, args)
  File "/usr/lib/python3/dist-packages/pip/_internal/commands/install.py", line 279, in run
    options.use_user_site = decide_user_install(
  File "/usr/lib/python3/dist-packages/pip/_internal/commands/install.py", line 630, in decide_user_install
    if site_packages_writable(root=root_path, isolated=isolated_mode):
  File "/usr/lib/python3/dist-packages/pip/_internal/commands/install.py", line 575, in site_packages_writable
    test_writable_dir(d) for d in set(get_lib_location_guesses(**kwargs))
  File "/usr/lib/python3/dist-packages/pip/_internal/commands/install.py", line 569, in get_lib_location_guesses
    scheme = distutils_scheme('', *args, **kwargs)
  File "/usr/lib/python3/dist-packages/pip/_internal/locations.py", line 111, in distutils_scheme
    obj = d.get_command_obj('install', create=True)
  File "/usr/lib/python3.8/distutils/dist.py", line 868, in get_command_obj
    self._set_command_options(cmd_obj, options)
  File "/usr/lib/python3.8/distutils/dist.py", line 910, in _set_command_options
    raise DistutilsOptionError(
distutils.errors.DistutilsOptionError: error in setup.cfg: command 'install' has no such option 'single_version_externally_managed'
`

How can I fix it?

**Comment by thenaterhood** [comment]
Edit setup.cfg and delete the line that
says single-version-externally-managed=1. I need to patch that (or feel
free to open a pull request).

--

**Comment by melodie11** [comment]
```
$ sudo pip3 install -e .
[sudo] Mot de passe de melodie : 
Obtaining file:///home/melodie/T%C3%A9l%C3%A9chargements/gscreenshot-2.12.1
Requirement already satisfied: pillow in /usr/lib/python3/dist-packages (from gscreenshot==2.12.1) (7.0.0)
Installing collected packages: gscreenshot
  Running setup.py develop for gscreenshot
Successfully installed gscreenshot
```
That's much better! Thank you!

" (or feel free to open a pull request)."

I am sure it will take you less time to patch it, than for me to find how to operate a pull request. Anyway thank you for this scrot gui! I have been using it for several years now!

**Comment by thenaterhood** [comment]
All good! And you're welcome :)

Fixed in 3de68249b9d54cd99c7f904fdae485cd2cf86913, and the fix will be in the next release.
